config/routes.rb

```diff
@@ -17,25 +17,20 @@
     post "/medical-equipment", to: "medical_equipment#submit"
 
     # Question 1.1: What kind of business are you?
-    # Question removed so redirect to next question, for users currently trying to access
-    get "/are-you-a-manufacturer", to: redirect(path: "medical-equipment-type")
-    post "/are-you-a-manufacturer", to: redirect(path: "medical-equipment-type")
+    # Question removed
```

Comment:
Nitpick: I don't think this line is needed, and the one about can be removed. The same goes for the product details and additional product questions.